ports-of-call/portability.hpp

```diff
@@ -53,6 +53,7 @@
 #define PORTABLE_MALLOC(size) Kokkos::kokkos_malloc<>(size)
 #define PORTABLE_FREE(ptr) Kokkos::kokkos_free<>(ptr)
 // Do we want to include additional terms here (for memory spaces, etc.)?
+#define PORTABLE_FENCE(name) Kokkos::fence(name)
```

Comment:
So you don't always need a name:
```suggestion
#define PORTABLE_FENCE(...) Kokkos::fence(__VA_ARGS__)
```